pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -103,9 +101,31 @@ func newPodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, handler netd
 		podInterfaceName: podInterfaceName,
 		iface:            correspondingNetworkIface,
 		launcherPID:      launcherPID,
+		dhcpConfigurator: dhcpConfigurator,
 	}, nil
 }
 
+func newBridgeDhcpConfigurator(iface v1.Interface, vmi v1.VirtualMachineInstance, podInterfaceName string, launcherPID *int, handler netdriver.NetworkHandler, cacheFactory cache.InterfaceCacheFactory) *dhcpconfigurator.DhcpConfigurator {
+	fakeBridgeIP, err := getFakeBridgeIP(iface.Name, vmi.Spec.Domain.Devices.Interfaces)
+	if err != nil {
+		return nil
+	}
+	fakeServerAddr, err := netlink.ParseAddr(fakeBridgeIP)
+	if err != nil || fakeServerAddr == nil {
+		return nil
+	}
+	bridgeIfaceName := fmt.Sprintf("k6t-%s", podInterfaceName)
```

Comment:
we can do this inside `NewBridgeDhcpServer` to put all the Bridge DHCP code there.

Reply:
I didn't do it because `getFakeBridgeIP` is locally available in this package. 

I could move it to an assorted helper file somewhere else though.

Reply:
we can have a `pkg/network/bridge/fake.go` also we may end up adding more stuff realted to bridge at `pkg/network/bridge/`